source/frontend/src/features/mapSideBar/router/ManagementFileActivityRouter.tsx

```diff
@@ -40,16 +40,40 @@ export const ManagementFileActivityRouter: React.FunctionComponent<
           <ManagementActivityEditContainer
             managementFileId={Number(match.params.managementFileId)}
             onClose={() => {
-              const parentPath = match?.url.substring(0, match?.url.lastIndexOf('/')) || '/';
+              const parentPath = generatePath(
```

Comment:
I'm guessing that these need to be updated as well based on our friday conversation